dask_expr/tests/test_concat.py

```diff
@@ -95,3 +95,33 @@ def test_concat_simplify_projection_not_added(pdf, df):
     assert result._name == expected._name
 
     assert_eq(q, pd.concat([pdf, pdf2])[["y", "x"]])
+
+
+def test_concat_axis_one_co_aligned(pdf, df):
+    df2 = df.rename({"x": "x_2", "y": "y_2"})
+    pdf2 = pdf.add_suffix("_2")
+    assert_eq(concat([df, df2], axis=1), pd.concat([pdf, pdf2], axis=1))
+
+
+def test_concat_axis_one_all_divisions_unknown(pdf, df):
```

Comment:
```suggestion
def test_concat_axis_one_all_divisions_unknown(pdf):
```
Looks like you aren't using the `df` fixture here.